src/i18n/locales/en.json

```
@@ -317,6 +317,17 @@
     "gemma-7b-it": "Gemma 7b",
     "gemma2-9b-it": "Gemma2 9b"
   },
+  "minimaxApi": {
```

Biome (2.4.7)

[error] The key minimaxApi was already declared.

(lint/suspicious/noDuplicateObjectKeys)